core/app/[locale]/(default)/cart/page-data.ts

```diff
@@ -194,3 +260,35 @@ export const getCart = async (variables: Variables) => {
 
   return data;
 };
+
+export const getShippingCountries = async (geography: FragmentOf<typeof GeographyFragment>) => {
+  const hasAccessToken = Boolean(process.env.BIGCOMMERCE_ACCESS_TOKEN);
+  const shippingZones = hasAccessToken ? await getShippingZones() : [];
+  const countries = geography.countries ?? [];
+
+  const uniqueCountryZones = shippingZones.reduce<string[]>((zones, item) => {
+    item.locations.forEach(({ country_iso2 }) => {
+      if (zones.length === 0) {
+        zones.push(country_iso2);
+
+        return zones;
+      }
+
+      const isAvailable = zones.length > 0 && zones.some((zone) => zone === country_iso2);
+
+      if (!isAvailable) {
+        zones.push(country_iso2);
+      }
+
+      return zones;
+    });
+
+    return zones;
+  }, []);
```

Comment:
🍹 I might try using a `Set` here to clean it up a bit.

Reply:
Good idea